package.json

```diff
@@ -71,35 +71,35 @@
     "prettier": "^1.12.1",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
-    "typescript": "^2.8.3"
+    "typescript": "^3.0.1"
   },
   "peerDependencies": {
-    "eslint": "<5 >=4.7.1",
+    "eslint": "<6 >5.0.0",
     "prettier": "^1.12.1"
   },
   "dependencies": {
-    "babel-eslint": "8.2.3",
-    "eslint-config-prettier": "2.9.0",
+    "babel-eslint": "^9.0.0",
+    "eslint-config-prettier": "^3.0.1",
     "eslint-module-utils": "2.1.1",
-    "eslint-plugin-ava": "4.5.1",
-    "eslint-plugin-babel": "4.1.2",
+    "eslint-plugin-ava": "^5.1.0",
+    "eslint-plugin-babel": "^5.1.0",
     "eslint-plugin-chai-expect": "1.1.1",
     "eslint-plugin-flowtype": "2.41.0",
-    "eslint-plugin-import": "2.9.0",
-    "eslint-plugin-jest": "21.14.1",
-    "eslint-plugin-jsx-a11y": "6.0.3",
+    "eslint-plugin-import": "^2.14.0",
```

Comment:
Any reason these version numbers were loosened?

Asking because I've found loose version numbers with `^` make new rules potentially hard to track (requires looking at `yarn.lock` or use `yarn why <dependency-name>` to spot the exact installed version).

Reply:
No particular reason here, just bumped via yarn without specifying the exact version. I'll pin the version - thanks 👍